apps/site/lib/site_web/controllers/schedule/timetable_controller.ex

```diff
@@ -22,13 +22,20 @@ defmodule SiteWeb.ScheduleController.TimetableController do
     as: :assign_direction_id
 
   def show(conn, _) do
+    direction_id = conn.assigns[:direction_id]
+    direction_name = conn.assigns.route.direction_names[direction_id]
+
+    {:ok, full_date} = Timex.format(conn.assigns.date, "{Mfull} {D}, {YYYY}")
```

Comment:
Could we call this `formatted_date` or something to distinguish it from a struct? Might be confusing with all the `date` related stuff floating around.